lib/src/reducers/dna_extensions_move_reducer.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:redux/redux.dart';
+import '../state/dna_extensions_move.dart';
+
+import '../actions/actions.dart' as actions;
+
+Reducer<DNAExtensionsMove> optimized_dna_extensions_move_reducer = combineReducers([
+  dna_extensions_move_reducer,
+]);
+
+Reducer<DNAExtensionsMove> dna_extensions_move_reducer = combineReducers([
+  TypedReducer<DNAExtensionsMove, actions.DNAExtensionsMoveSetSelectedExtensions>(
+      dna_extensions_move_set_selected_extensions_reducer),
+  TypedReducer<DNAExtensionsMove, actions.DNAExtensionsMoveAdjustPosition>(
+      dna_extensions_move_adjust_reducer),
+  TypedReducer<DNAExtensionsMove, actions.DNAExtensionsMoveStop>(dna_extensions_move_stop_reducer),
+]);
+
+DNAExtensionsMove dna_extensions_move_set_selected_extensions_reducer(
```

Comment:
A better name is `dna_extensions_move_set_selected_extension_ends_reducer` (for similar reasons to other similar rename requests)